provisioning/dashboards/ciphertrust-manager-kmip-statistics.json

```diff
@@ -0,0 +1,501 @@
+          "hide": false,
+          "interval": "",
+          "legendFormat": "MacV",
+          "refId": "V"
+        },
+        {
+          "exemplar": true,
+          "expr": "sum(rate(ciphertrust_nae_kmip_operation_success{instance =~ \"$instances\", job=~\"$job\", operation =~ \"ReKey\"}[$__rate_interval]) + rate(ciphertrust_nae_kmip_operation_failure{instance =~ \"$instances\", job=~\"$job\", operation =~ \"ReKey\"}[$__rate_interval]))",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "ReKey",
+          "refId": "W"
+        },
+        {
+          "exemplar": true,
+          "expr": "sum(rate(ciphertrust_nae_kmip_operation_success{instance =~ \"$instances\", job=~\"$job\", operation =~ \"ReKey Key Pair\"}[$__rate_interval]) + rate(ciphertrust_nae_kmip_operation_failure{instance =~ \"$instances\", job=~\"$job\", operation =~ \"ReKey Key Pair\"}[$__rate_interval]))",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "ReKey Key Pair",
+          "refId": "X"
+        }
+      ],
+      "title": "Rate of Operation",
+      "type": "timeseries"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 32,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": null,
+        "definition": "label_values(instance)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "instances",
+        "multi": true,
+        "name": "instances",
+        "options": [],
+        "query": {
+          "query": "label_values(instance)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 5,
+        "type": "query"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "text": [
+            "CipherTrust Manager"
+          ],
+          "value": [
+            "CipherTrust Manager"
+          ]
+        },
+        "datasource": null,
+        "definition": "label_values(job)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "job",
+        "multi": true,
+        "name": "job",
+        "options": [],
+        "query": {
+          "query": "label_values(job)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Cipher Trust Manager KMIP Statistics",
```

Comment:
Should be "CipherTrust Manager" (there was an extra space).

Reply:
Updated the title and renamed the file.